templates/central-config.yaml

```diff
@@ -0,0 +1,98 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "xl-deploy.fullname" . }}-central-config
+  labels:
+    chart: {{ template "xl-deploy.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.centralConfiguration.replicas }}
+  selector:
+    matchLabels:
+      app: {{ template "xl-deploy.name" . }}
+      component: {{ template "xl-deploy.name" . }}-central-config
+      release: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "xl-deploy.name" . }}
+        component: {{ template "xl-deploy.name" . }}-central-config
+        release: {{ .Release.Name }}
+    spec:
+      securityContext:
+        fsGroup: 10001
+      {{- if .Values.centralConfiguration.image.pullSecret }}
+      imagePullSecrets:
+        - name: {{ .Values.centralConfiguration.image.pullSecret }}
```

Comment:
Openshift allows pulling secrets through service accounts, not like this. Though it may work.

Reply:
I deployed in local CRC and everything was working fine. But I see we are using serviceAccount for the other pods. Will use the same here.

Reply:
Updated based on other pod definitions